Add content from community doc PR:1105, 1109 to 1111 (#213)
rancher/longhorn-product-docs: docs/version-1.9.0/modules/en/pages/longhorn-system/settings.adoc

* `replica-data`: Represents replica data store.
* `instance`: Represents engine and replica runtime instance.

=== Orphaned Resource Automatic Deletion Grace Period

*Default value*: `300` seconds

> Specifies the wait time before {longhorn-product-name} automatically deletes an orphaned Custom Resource (CR) and the actual orphaned data or runtime instance it represents.

Reviewer: ```suggestion
Number of seconds {longhorn-product-name} waits before automatically deleting an orphaned custom resource (CR) and the actual orphaned data or runtime instance it represents.
```

[NOTE]
====

> If a user manually deletes an orphaned CR, the deletion occurs immediately and does not respect this grace period.

Reviewer: ```suggestion
The grace period does not take effect when you manually delete an orphaned CR.
```
